resources/lang/fr.json

```diff
@@ -348,17 +348,17 @@
     "Communication disabilities": "Troubles de la communication",
     "Communications": "Communications",
     "Communities served": "Communautés desservies",
-    "Communities we :represent_or_serve_and_support": "Communautés que nous :represent_or_serve_and_support",
+    "Communities we :represent_or_serve_and_support": "Communautés que nous représentons",
```

Comment:
Depending on the context, `:represent_or_serve_and_support` is replaced by the translations for `__('represent')` or `__('serve and support')`. It's likely that the placeholder should remain.

Reply:
Thanks @jobara I will go ahead and revert it.
I also excluding language files from Prettier since it will remove `\/` from strings when it formats the file.